---
layout: page
title: Projects
sidebar_link: true
sidebar_sort_order: 3
---


> ## Epigenetic Gene Regulation Model
<img src="assets/img/CellLa.jpg" style="width:800px; height:500px;" />
<!--
![placeholder](/assets/img/CellLa.jpg/800x400 "CellLa")
-->
The Multi-Attention based Deep Learning model aims to learn regulatory latent space of epigenetic/transcriptional markers such as histone modifications, methylations, and transcription factors.
* Embedding different epigenetic/transcriptional markers into a regulatory latent space using marker-specific architectures such as CNN & RNN with attention (histone modification), RNN with attention (methylation), self-attention network (transcription factor).
* Integrating each latent feature by multi-attention block to interpret relationships between multiple regulatory features.
* In experiments with 18 cell lines, the proposed model predicted the gene expression level more accurately than the state-of-the-art model.
* Reveal the cell-type-specific gene regulation mechanisms and enriched genes in terms of their functions and epigenetic regulation.

> ## Pathway Attention Model

> ## RKSS Kernel

> ## MIDAS
