---
layout: page
title: Publications
sidebar_link: true
sidebar_sort_order: 2
---
## In Review (\*: Equal contributors)
- [1] <strong>Lee S</strong>, Piao Y, Lee D, Kim S. Multi-Task Informed Learnable Prototypes on Few Shot learning for Molecular Property Prediction
- [2] <strong>Lee S*</strong>, Park J\*, Piao Y, Lee D, Lee D, Kim S. Multi-layered Knowledge Graph Neural Network Reveals Pathway-level Agreement of Three Breast Cancer Multi-gene Assays
- [3] Cho C*, <strong>Lee S*</strong>, Piao Y, Bang D, Kim S. ChemAP: Chemical Structure-Based Deep Learning Model via Knowledge Distillation for Predicting Drug Approval before Clinical Trial Phase
- [4] Sung I*, <strong>Lee S*</strong>, Bang D, Yi J, Kim S. MDTR: A Knowledge-Guided Interpretable Representation for Quantifying Liver Toxicity at Transcriptomic Level
- [5] Shin Y*, <strong>Lee S*</strong>, Pak M, Cho C, Kim S. DrugPT-Net: Drug Perturbation Guided Visible Neural Network for Drug Response Prediction at Transcriptomic Level
- [6] Lu Y*, <strong>Lee S*</strong>, Park S, Yi J, Cho C, Lim S, Kang S, Kim S. An ensemble strategy to improve generalization power of deep learning models for DTI prediction
- [7] Lu Y, Piao Y, <strong>Lee S</strong>, Kim S. Context-Aware Hierarchical Fusion for Drug Relational Learning


## International Journals/Conferences (\*: Equal contributors)
- [1] Piao Y, Lee S, Lu Y, Kim S. Improving Out-of-Distribution Generalization in Graphs vis Hierarchical Semantic Environments. CVPR 2024. Acceted. Jun. 17, 2024.
- [[2] Park S*, <strong>Lee S*</strong>, Pak M, Kim S. Dual Representation Learning for Predicting Drug-side Effect Frequency using Protein Target Information. <em>IEEE Journal of Biomedical and Health Informatics</em>. Jan. 5, 2024.](https://ieeexplore.ieee.org/document/10381882/)
- [[3] Yi J, <strong>Lee S</strong>, Lim S, Cho C, Piao Y, Yeo M, Kim D, Kim S, Lee S. Exploring chemical space for lead identification by propagating on chemical similarity network. <em>Computational and Structural Biotechnology Journal</em>. Aug. 25, 2023; 21:4187-4195.](https://www.sciencedirect.com/science/article/pii/S200103702300291X)
- [[4] Gu J\*, Bang D\*, Yi J\*, <strong>Lee S</strong>, Kim DK, Kim S. A model-agnostic framework to enhance knowledge graph-based drug combination prediction with drug–drug interaction data and supervised contrastive learning. <em>Briefings in Bioinformatics</em>. Aug. 7, 2023; 24(5):bbad285.](https://academic.oup.com/bib/article/24/5/bbad285/7237941)
- [[5] Bang D, Lim S, <strong>Lee S</strong>, Kim S. Biomedical knowledge graph learning for drug repurposing by extending guilt-by-association to multiple layers. <em>Nature Communications</em> (SCIE, IF=16.6). Jun. 15, 2023; 14(1):3570.](https://www.nature.com/articles/s41467-023-39301-y)
- [[6] Pak M*, <strong>Lee S*</strong>, Sung I, Koo B, Kim S. Improved drug response prediction by drug target data integration via network-based profiling. <em>Briefings in Bioinformatics</em>. Feb. 8, 2023; 24(2):bbad034.](https://academic.oup.com/bib/article/24/2/bbad034/7031154)
- [[7] <strong>Lee S</strong>, Lee D, Piao Y, Kim S. SPGP: Structure Prototype Guided Graph Pooling. <em>NeurIPS 2022 Workshop New Frontiers in Graph Learning (GLFrontiers)</em>. Dec. 2, 2022.](https://arxiv.org/abs/2209.07817)
- [[8] Koo B, Lee D, <strong>Lee S</strong>, Sung I, Kim S, Lee S. Risk Stratification for Breast Cancer Patient by Simultaneous Learning of Molecular Subtype and Survival Outcome Using Genetic Algorithm-Based Gene Set Selection. <em>Cancers</em>. \[From The 10th International Conference on Intelligent Biology and Medicine (ICIBM 2022)\]. Aug. 25, 2022; 14(17):4120.](https://www.mdpi.com/2072-6694/14/17/4120)
- [[9] Lim S*, <strong>Lee S*</strong>, Piao Y, Choi MG, Bang D, Gu J, Kim S. On Modeling and Utilizing Chemical Compound Information with Deep Learning Technologies: A Task-oriented Approach. <em>Computational and Structural Biotechnology Journal</em>. Aug. 4, 2022; 20:4288-4304.](https://www.sciencedirect.com/science/article/pii/S2001037022003300)
- [[10] Moon JH, <strong>Lee S</strong>, Pak M, Hur B, Kim S. MLDEG: A Machine Learning Approach to Identify Differentially Expressed Genes Using Network Property and Network Propagation. <em>IEEE/ACM Transactions on Computational Biology and Bioinformatics</em>. Jul.-Aug., 2022; 19(4).](https://ieeexplore.ieee.org/document/9382840)
- [[11] Piao Y, <strong>Lee S</strong>, Lee D, Kim S. Sparse Structure Learning via Graph Neural Networks for Inductive Document Classification. <em>AAAI 2022</em>. Feb. 22, 2022.](https://arxiv.org/abs/2112.06386)
- [[12] Sung I*, <strong>Lee S*</strong>, Pak M, Shin Y, Kim S. AutoCoV: Learning the Spatial and Temporal Dynamics of COVID-19 Spread Patterns from Embedding Space by k-mer Based Deep Learning. <em>BMC Bioinformatics</em>. [From The 10th International Conference on Intelligent Biology and Medicine (ICIBM 2022)]. Apr., 2022; 23(3):149.](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-022-04679-x)
- [[13] Kim J, Lim S, <strong>Lee S</strong>, Cho C, Kim S. Embedding of FDA Approved Drugs in Chemical Space Using Cascade Autoencoder with Metric Learning. <em>IEEE BigComp 2022. AI-BioHealth 2022 workshop</em>. Jan. 17, 2022](https://ieeexplore.ieee.org/document/9736489)
- [[14] Kim I, <strong>Lee S</strong>, Kim Y, Namkoong H, Kim S. Probabilistic model for pathway-guided gene set selection. <em>IEEE International Conference on Bioinformatics and Biomedicine (BIBM) 12th International Workshop on Biomedical and Health Informatics (BHI 2021)</em>. Dec. 9, 2021.](https://ieeexplore.ieee.org/document/9669350)
- [[15] Kim MS, <strong>Lee S</strong>, Lim S, Lee DY, Kim S. Subnetwork Representation Learning for Discovering Network Biomarkers in Predicting Lymph Node Metastasis in Early Oral Cancer.. <em>Scientific Reports</em>. Dec., 2021; 11:23992.](https://www.nature.com/articles/s41598-021-03333-5)
- [[16] Lee T, <strong>Lee S</strong>, Kang M, Sun Kim. Deep Hierarchical Embedding for Simultaneous Modeling of GPCR Proteins in a Unified Metric Space. <em>Scientific Reports</em>. May., 2021; 11:9543.](https://www.nature.com/articles/s41598-021-88623-8)
- [[17] <strong>Lee S</strong>, Lee T, Noh YK, Kim S. Ranked k-spectrum kernel for comparative and evolutionary comparison of exons, introns, and CpG islands. <em>IEEE/ACM Transactions on Computational Biology and Bioinformatics</em>. May.-Jun., 2021; 18(30).](https://ieeexplore.ieee.org/abstract/document/8823008)
- [[18] Jeong D, Lim S, <strong>Lee S</strong>, Oh M, Cho C, Seong H, Jung W, Kim S. Construction of Condition-Specific Gene Regulatory Network using Kernel Canonical Correlation Analysis. <em>Frontiers in Genetics</em>. May., 2021; 12:652623.](https://www.frontiersin.org/articles/10.3389/fgene.2021.652623/abstract)
- [[19] Park YJ, <strong>Lee S\*</strong>, Lim S*, Hahn N, Ji Y, Huh JY, Alfadda AA, Kim S, Kim JB. DNMT1 maintains metabolic fitness of adipocytes through acting as an epigenetic safeguard of mitochondrial dynamics. <em>Proceedings of the National Academy of Sciences (PNAS)</em>. Mar. 16, 2021; 118(11):e 2021073118.](https://www.pnas.org/content/118/11/e2021073118)
- [[20] Oh M, Park S, <strong>Lee S</strong>, Lee D, Lim S, Jeong D, Jo K, Jung I, Kim S. DRIM: A web-based system for investigating drug response at the molecular level by condition-specific multi-omics data integration. <em>Frontiers in Genetics</em>. Nov., 2020; 11:564792.](https://www.frontiersin.org/articles/10.3389/fgene.2020.564792/full)
- [[21] Kang M\*, <strong>Lee S\*</strong>, Lee D, Kim S. Learning Cell-Type-Specific Gene Regulation Mechanisms by Multi-Attention Based Deep Learning with Regulatory Latent Space. <em>Frontiers in Genetics</em>. Sep. 30, 2020; 11:869.](https://www.frontiersin.org/articles/10.3389/fgene.2020.00869/full)
- [[22] <strong>Lee S</strong>, Lim S, Lee T, Sung I, Kim S. Cancer subtype classification and modeling by pathway attention and propagation. <em>Bioinformatics</em>. Jun. 15, 2020; 36(12):3818-3824.](https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btaa203/5811233)
- [[23] Lim S, <strong>Lee S</strong>, Jung I, Rhee S, Kim S. Comprehensive and critical evaluation of individualized pathway activity measurement tools on pan-cancer data. <em>Briefings in Bioinformatics</em>. Jan., 2020; 21(1):36-46.](https://academic.oup.com/bib/article/21/1/36/5181542)
- [[24] Hur B, Kang D, <strong>Lee S</strong>, Moon JH, Lee G, Kim S. Venn-diaNet: Venn diagram based network propagation analysis framework for computing multiple biological experiments. <em>BMC Bioinformatics</em> \[From Proceeding of 30th International Conference on Genome Informatics (GIW)\]. Dec. 27, 2019; 20(23):1-12.](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-019-3302-7)
- [[25] Kang D, Ahn H, <strong>Lee S</strong>, Lee CJ, Hur J, Jung W, Kim S. StressGenePred: a twin prediction model architecture for classifying the stress types of samples and discovering stress-related genes in Arabidopsis. <em>BMC Genomics</em> \[From Proceeding of 2018 IEEE International Conference on Bioinformatics and Biomedicine (BIBM)\]. Dec. 20, 2019; 20(11):949.](https://link.springer.com/article/10.1186/s12864-019-6283-z)
- [[26] Kim MS, <strong>Lee S</strong>, Lim S, Kim S. SpliceHetero: An information theoretic approach for measuring spliceomic intratumor heterogeneity from bulk tumor RNA-seq. <em>Plos ONE</em>. Oct. 23, 2019; 14(10).](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0223520)
- [[27] Lee DH. <strong>Lee S</strong>, Kim S. PRISM: Methylation Pattern-based, Reference-free Inference of Subclonal Makeup. <em>Bioinformatics</em> \[From Proceeding of 27th conference on Intelligent Systems for Molecular Biology (ISMB)\]. Jul. 5, 2019; 35(14):i520-i529.](https://academic.oup.com/bioinformatics/article/35/14/i520/5529252)
- [[28] Lee CJ, Kang D, <strong>Lee S</strong>, Lee S, Kang J, Kim S. In silico experiment system for testing hypothesis on gene functions using three condition specific biological networks. <em>Methods</em>. May. 25, 2018; 145:10-15.](https://www.sciencedirect.com/science/article/pii/S104620231830001X)
- [[29] <strong>Lee S</strong>, Park Y, Kim S. MIDAS: Mining differentially activated subpaths of KEGG pathways from multi-class RNA-seq data. <em>Methods</em>. Jul. 15, 2017; 124:13-24.](https://www.sciencedirect.com/science/article/pii/S1046202317300488)
- [[30] Moon JH, Lim S, Jo K, <strong>Lee S</strong>, Seo S, Kim S. PINTnet: construction of condition-specic pathway interaction network by computing shortest paths on weighted PPI. <em>BMC Systems Biology</em> \[From Proceeding of 15th Asia Pacific Bioinformatics Conference (APBC)\]. Mar. 14, 2017; 11(2):15.](https://link.springer.com/article/10.1186/s12918-017-0387-3)
- [[31] <strong>Lee S</strong>, Moon JH, Park Y, Kim S. Flow maximization analysis of cell cycle pathway activation status in breast cancer subtypes. <em>IEEE International Conference on Big Data and Smart Computing (BigComp) 2017</em>. Feb. 13-16, 2017. Jeju, Korea.](https://ieeexplore.ieee.org/abstract/document/7881721)
- [[32] Chae H, <strong>Lee S</strong>, Nephew KP, Kim S, Subtype-specific CpG island shore methylation and mutation patterns in 30 breast cancer cell lines. <em>BMC Systems Biology</em> \[From Proceeding of 27th International Conference on Genome Informatics (GIW)\]. Dec. 23, 2016; 10(4):116.](https://link.springer.com/article/10.1186/s12918-016-0356-2)
- [[33] Chae H, <strong>Lee S</strong>, Seo S, Jung D, Chang H, Nephew KP, Kim S. BioVLAB-mCpG-SNP-EXPRESS: A system for multi-level and multi-perspective analysis and exploration of DNA methylation, sequence variation (SNPs), and gene expression from multi-omics data. <em>Methods</em>. Jul. 28, 2016; 111:64-71.](https://www.sciencedirect.com/science/article/pii/S1046202316302304)
- [[34] Jeong HM, <strong>Lee S</strong>, Chae H, Kim R, Kwon MJ, Oh E, Choi YL, Kim S, Shin YK. Efficiency of methylated DNA immunoprecipitation bisulphite sequencing for whole-genome DNA methylation analysis. <em>Epigenomics</em>. Jun. 8, 2016; 8(8):1061-1077.](https://www.futuremedicine.com/doi/abs/10.2217/epi-2016-0038)


## Book Chapters
- [1] Pak M, Jeong D, Moon JH, Ann H, Hur B, <strong>Lee S</strong>, Kim S. Network Propagation for the Analysis of Multi-Omics Data. Recent Advances in Biological Network Analysis (pp. 185-217). 2021

## Domestic Journal (*: equal contributors)
- [1] 조창연\*, <strong>이상선\*</strong>, 김선. 그래프 기반 Positive-Unlabeled 학습 모델의 앙상블 체계를 사용한 철회 약물 예측. 한국정보과학회 2022 한국컴퓨터종합학술대회 논문집, Vol.49, pp.777-779. Jun., 2022.
- [2] 박민우, <strong>이상선</strong>, 성인영, 신윤열, 정인욱, 김선. 밀도기반 군집화와 딥러닝 모델을 이용한 COVID-19 바이러스 전장 유전체 임베딩 전략. Journal of KIISE \[From 2021 한국컴퓨터종합학술대회\] 제49권 제4호, pp.261-270. 2022.
- [3] 박은화\*, <strong>이상선\*</strong>, 김선. 신약개발과 맞춤의학을 위한 그래프 학습 기술. 한국정보과학회지 제 40권 제 3호, pp.30-40. 2022.
- [4] 성인영, <strong>이상선</strong>, 이도훈, 김선. 네트워크 전파 및 랜덤 포레스트를 활용한 유방암 환자 생존 분석. 한국정보과학회 2021 한국컴퓨터종합학술대회 논문집, Vol.48, pp.609-611. Jun., 2021.
- [5] 박은화, <strong>이상선</strong>, 김선. 헬스케어 예측을 위한 전자 건강 기록 기반 그래프 컨볼루션 모델. 한국정보과학회 2020 한국컴퓨터종합학술대회 논문집, Vol.47, pp.569-571. Jul., 2020.
- [6] 강동원, 안홍렬, <strong>이상선</strong>, 정우석, 김선. 시계열 유전자 발현 데이터의 스트레스 특이 반응 유전자 검출 및 스트레스 예측 알고리즘 개발. 한국정보과학회 2018 한국컴퓨터종합학술대회 논문집, Vol.45, pp.1063-1065. Jun., 2018.
