<!DOCTYPE html>
<html lang="en-us">

  <head>
  <link href="https://gmpg.org/xfn/11" rel="profile" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <meta http-equiv="content-type" content="text/html; charset=utf-8" />

  <!-- Enable responsiveness on mobile devices-->
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1" />

  <title>
    
      Publications &middot; Sangseon Lee
    
  </title>

  


  <!-- CSS -->
  <link rel="stylesheet" href="/assets/css/main.css" />
  

<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Abril+Fatface" />

  <!-- Icons -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="/favicon.png" />
<link rel="shortcut icon" href="/favicon.ico" />

  <!-- RSS -->
  <link rel="alternate" type="application/rss+xml" title="RSS" href="/feed.xml" />

  <!-- Additional head bits without overriding original head -->
</head>


  <body class="page">

    <div id="sidebar">
  <header>
    <img src="assets/img/Sangseon_Lee.jpg" alt="Sangseon_Lee" style="display: block; margin-left: auto; margin-right: auto; width: 50%">
    <div class="site-title">
      <a href="/">
        
          <span class="back-arrow icon"><svg fill="#000000" height="24" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg">
  <path d="M0 0h24v24H0z" fill="none"/>
  <path d="M20 11H7.83l5.59-5.59L12 4l-8 8 8 8 1.41-1.41L7.83 13H20v-2z"/>
</svg></span>
        
        Sangseon Lee
      </a>
    </div>
    <p class="lead">Ph.D. in Computer Science & Engineering</p>
  </header>
  <nav id="sidebar-nav-links">
  <!-- 
    <a class="home-link "
        href="/">Biography</a>
   -->
  

  

  


  

  
    
  

  
    
  

  

  

  

  
    
  

  
    
      <a class="page-link "
          href="/">Biography</a>
    
  

  
    
      <a class="page-link  active"
          href="/publications.html">Publications</a>
    
  

  
    
      <a class="page-link "
          href="/projects.html">Projects</a>
    
  


  


  

  
    
  

  
    
  

  

  

  

  
    
  

  
    
  

  
    
  

  
    
  


  <!-- Optional additional links to insert in sidebar nav -->
</nav>


  

  <nav id="sidebar-icon-links">
  

  

  <a id="googlescholar-link"
     class="icon" title="Google Scholar" aria-label="Google Scholar"
     href="https://scholar.google.co.kr/citations?hl=en&user=abgnGj0AAAAJ">
    <svg role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><title>Google Scholar icon</title><path d="M10.93 2.045c-.547.366-3.22 2.14-5.938 3.945C2.272 7.794.05 9.286.05 9.304c0 .019.136.11.305.2.167.096 2.85 1.583 5.965 3.31l5.656 3.143.144-.074c.082-.04 2.169-1.232 4.642-2.642l4.493-2.568.027 7.947h2.668V9.319l-3.46-2.32c-4.664-3.124-8.392-5.586-8.484-5.606-.045-.008-.527.287-1.076.652M5.355 16.633l.014 2.005 3.31 1.987 3.31 1.982 3.337-2 3.332-2.005V16.62c0-1.092-.013-1.983-.027-1.983s-1.318.782-2.9 1.741l-3.306 1.996-.431.256-1.32-.791a604.12 604.12 0 0 1-3.286-1.979l-2.005-1.21c-.024-.008-.032.883-.027 1.983"/></svg>
  </a>

  <a id="orcid-link"
     class="icon" title="ORCID" aria-label="ORCID"
     href="https://orcid.org/0000-0002-7398-9580">
    <svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>ORCID icon</title><path d="M12 0C5.372 0 0 5.372 0 12s5.372 12 12 12 12-5.372 12-12S18.628 0 12 0zM7.369 4.378c.525 0 .947.431.947.947s-.422.947-.947.947a.95.95 0 0 1-.947-.947c0-.525.422-.947.947-.947zm-.722 3.038h1.444v10.041H6.647V7.416zm3.562 0h3.9c3.712 0 5.344 2.653 5.344 5.025 0 2.578-2.016 5.025-5.325 5.025h-3.919V7.416zm1.444 1.303v7.444h2.297c3.272 0 4.022-2.484 4.022-3.722 0-2.016-1.284-3.722-4.097-3.722h-2.222z"/></svg>
  </a>

  
  
  
  

  

  
    <a id="search-link"
       class="icon"
       title="Search" aria-label="Search"
       href="/search.html">
      <svg fill="#000000" height="24" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg">
    <path d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"/>
    <path d="M0 0h24v24H0z" fill="none"/>
</svg>
    </a>
  

  <!-- Optional additional links to insert for icons links -->
</nav>

  <p>
  &copy; 2020.
  <a href="/LICENSE.md">MIT License.</a>
</p>

</div>


    <main class="container">
      <header>
  <h1 class="page-title">Publications</h1>
</header>
<div class="content">
  <h2 id="international-journals">International Journals</h2>
<ul>
  <li><strong>Lee S</strong>, Lim S, Lee T, Sung I, Kim S. Cancer subtype classification and modeling by pathway attention and propagation. <em>Bioinformatics</em>, accepted.</li>
  <li>Lim S, <strong>Lee S</strong>, Jung I, Rhee S, Kim S. Comprehensive and critical evaluation of individualized pathway activity measurement tools on pan-cancer data. <em>Briefings in Bioinformatics</em>, 2020 Jan; 21(1):36-46.</li>
  <li>Hur B, Kang D, <strong>Lee S</strong>, Moon JH, Lee G, Kim S. Venn-diaNet: Venn diagram based network propagation analysis framework for computing multiple biological experiments. <em>BMC Bioinformatics</em>, 2019 Dec 27; 20(23):1-12.</li>
  <li>Kang D, Ahn H, <strong>Lee S</strong>, Lee CJ, Hur J, Jung W, Kim S. StressGenePred: a twin prediction model architecture for classifying the stress types of samples and discovering stress-related genes in Arabidopsis. <em>BMC Genomics</em>, 2019 Dec 20; 20(11):949.</li>
  <li>Kim MS, <strong>Lee S</strong>, Lim S, Kim S. SpliceHetero: An information theoretic approach for measuring spliceomic intratumor heterogeneity from bulk tumor RNA-seq. <em>Plos ONE</em>, 2019 Oct 23; 14(10).</li>
  <li><strong>Lee S</strong>, Lee T, Noh YK, Kim S. Ranked k-spectrum kernel for comparative and evolutionary comparison of exons, introns, and CpG islands. <em>IEEE/ACM Transactions on Computational Biology and Bioinformatics</em>, 2019 Sep 3, in press.</li>
  <li>Lee DH. <strong>Lee S</strong>, Kim S. PRISM: Methylation Pattern-based, Reference-free Inference of Subclonal Makeup. <em>Bioinformatics</em>, 2019 Jul 5; 35(14):i520-i529.</li>
  <li>Lee CJ, Kang D, <strong>Lee S</strong>, Lee S, Kang J, Kim S. In silico experiment system for testing hypothesis on gene functions using three condition specific biological networks. <em>Methods</em>, 2018 May 25; 145:10-15.</li>
  <li><strong>Lee S</strong>, Park Y, Kim S. MIDAS: Mining differentially activated subpaths of KEGG pathways from multi-class RNA-seq data. <em>Methods</em>, 2017 Jul 15; 124:13-24.</li>
  <li>Moon JH, Lim S, Jo K, <strong>Lee S</strong>, Seo S, Kim S, PINTnet: construction of condition-specic pathway interaction network by computing shortest paths on weighted PPI. <em>BMC Systems Biology</em>, 2017 Mar 14; 11(2):15.</li>
  <li>Chae H, <strong>Lee S</strong>, Nephew KP, Kim S, Subtype-specific CpG island shore methylation and mutation patterns in 30 breast cancer cell lines. <em>BMC Systems Biology</em>, 2016 Dec 23; 10(4):116.</li>
  <li>Chae H, <strong>Lee S</strong>, Seo S, Jung D, Chang H, Nephew KP, Kim S. BioVLAB-mCpG-SNP-EXPRESS: A system for multi-level and multi-perspective analysis and exploration of DNA methylation, sequence variation (SNPs), and gene expression from multi-omics data. <em>Methods</em>, 2016 Jul 28; 111:64-71.</li>
  <li>Jeong HM, <strong>Lee S</strong>, Chae H, Kim R, Kwon MJ, Oh E, Choi YL, Kim S, Shin YK. Efficiency of methylated DNA immunoprecipitation bisulphite sequencing for whole-genome DNA methylation analysis. <em>Epigenomics</em>, 2016 Jun 8; 8(8):1061-1077.</li>
</ul>

<h2 id="international-conferences">International Conferences</h2>
<ul>
  <li>Hur B, Kang D, <strong>Lee S</strong>, Moon JH, Lee G, Kim S. Venn-diaNet: Venn diagram based network propagation analysis framework for computing multiple biological experiments. <em>30<sup>th</sup> International Conference on Genome Informatics (GIW) 2019</em>, Dec 9-11, Sydney, Australia.</li>
  <li>Lee DH. <strong>Lee S</strong>, Kim S. PRISM: Methylation Pattern-based, Reference-free Inference of Subclonal Makeup. <em>27<sup>th</sup> conference on Intelligent Systems for Molecular Biology (ISMB) 2019</em>, July 21-25, Basel, Switzerland.</li>
  <li>Kang D, Ahn H, <strong>Lee S</strong>, Lee CJ, Hur J, Jung W, Kim S. Identifying stress-related genes and predicting stress types in Arabidopsis using logical correlation layer and CMCL loss through time-series data. <em>IEEE International Conference on Bioinformatics and Biomedicine (BIBM) 2018</em>, Dec 3-6, Madrid, Spain.</li>
  <li><strong>Lee S</strong>, Moon JH, Park Y, Kim S, Flow maximization analysis of cell cycle pathway activation status in breast cancer subtypes, <em>IEEE International Conference on Big Data and Smart Computing (BigComp) 2017</em>, Feb 13-16, Jeju, Korea.</li>
  <li>Moon JH, Lim S, Jo K, <strong>Lee S</strong>, Seo S, Kim S, PINTnet: construction of condition-specic pathway interaction network by computing shortest paths on weighted PPI. <em>15<sup>th</sup> Asia Pacific Bioinformatics Conference (APBC) 2017</em>, Jan 16-18, Shenzhen, China.</li>
  <li>Chae H, <strong>Lee S</strong>, Nephew KP, Kim S, Subtype-specific CpG island shore methylation and mutation patterns in 30 breast cancer cell lines. <em>27<sup>th</sup> International Conference on Genome Informatics (GIW) 2016</em>, Oct 3-5, Shanghai, China.</li>
</ul>

<h2 id="in-progress--equal-contributors">In Progress (*: Equal contributors)</h2>
<ul>
  <li>Kang M*, <strong>Lee S*</strong>, Lee D, Kim S. Learning Cell-Type-Specific Gene Regulation Mechanisms by Multi-Attention Based Deep Learning with Regulatory Latent Space. In preparation for <em>Frontiers in Genetics</em>.</li>
  <li>Moon JH, <strong>Lee S</strong>, Hur B, Kim S. MLDEG: A network-based ensemble model to identify differentially expressed genes consistently. In preparation for <em>IEEE/ACM Transactions on Computational Biology and Bioinformatics.</em></li>
  <li>Lee T, <strong>Lee S</strong>, Kang M, Sun Kim. Simultaneous Modelling of Hierarchy in GPCR Proteins with Deep Learning in a Single Metric Space. In preparation for <em>European Conference on Computational Biology (ECCB) 2020</em>.</li>
  <li>Kim MS, <strong>Lee S</strong>, Lim S, Lee DY, Kim S. AttentiveCancer: an attention framework using subnetwork level representation of transcriptome for predicting lymph node metastasis in early oral cancer. In preparation for <em>European Conference on Computational Biology (ECCB) 2020</em>.</li>
</ul>

</div>

    </main>

    <!-- Optional footer content -->

  </body>
</html>
